### ISSUE TITLE
[BUG] Small typo in CLI docs

### ISSUE THREAD

**Issue by vlade11115** [schemathesis/schemathesis#845]
**Describe the bug**
On section https://schemathesis.readthedocs.io/en/stable/cli.html#how-are-responses-checked
```
To make Schemathesis perform all built-in checks use --check all CLI option:

$ schemathesis run --check all http://api.com/swagger.json
```
But, in real CLI:
```
schemathesis run --check all http://api.com/swagger.json
Usage: schemathesis run [OPTIONS] SCHEMA
Try 'schemathesis run -h' for help.

Error: no such option: --check  Did you mean --checks?
```
**To Reproduce**
1. Run this command `schemathesis run --check all http://api.com/swagger.json`
2. See error `Error: no such option: --check  Did you mean --checks?`

If possible, please post a minimal version of your API schema that cause this behavior:

```
pip freeze | grep schemathesis                      
schemathesis==2.7.1
```

**Expected behavior**
I except docs to contain command `schemathesis run --checks all http://api.com/swagger.json`, with **s** on the end.

I will make PR for this fix after we confirm this bug.

**Comment by Stranger6667** [comment]
Resolved via #846

**Comment by Stranger6667** [comment]
Thank you for reporting this :)
